packaging/common/cfengine-hub/preinstall.sh

```diff
@@ -145,51 +145,66 @@ then
   fi
 fi
 
-#
-# We check if there is a postgres db server running already
-#
-PSQL_RUNNING=`filter_netstat_listen ":5432\s"`
-if [ ! -z "$PSQL_RUNNING" ];
-then
-  cf_console echo "There seems to be a server listening on port 5432"
-  cf_console echo "This might mean that there is a PostgreSQL server running on the machine already"
-  cf_console echo "Checking if the Postgres installation belongs to a previous CFEngine deployment"
-  PSQL_COMMAND=$(ps -p $(fuser -n tcp 5432 2>/dev/null) -o args=|cut -d' ' -f1)
-  if [ ! -z "$PSQL_COMMAND" ];
+ensure_postgres_terminated() {
+  PSQL_RUNNING=`filter_netstat_listen ":5432\s"`
+  if [ ! -z "$PSQL_RUNNING" ];
```

Comment:
you could also invert this condition and write something like
```suggestion
  if [ -z "$PSQL_RUNNING" ];
  then
    exit 0
  fi
```
and remove one level of indentation from the following commands.

Less indentation - less disk usage - greener environment! :leaves:

Reply:
Good point. Changing.